src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,128 @@
+---
+linkTitle: Creating an app catalog
+title: Creating an app catalog
+description: How to create a custom app catalog for use with app platform and push helm charts to it.
+weight: 40
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-12-01
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I create an app catalog for my apps?
+  - How can I serve an app catalog using GitHub Pages?
+  - How can I create a catalog CR for my app catalog?
+  - How can I push an app to my app catalog?
+---
+
+# Creating an app catalog
+
+## Overview
+
+An app catalog is a collection of apps that can be deployed using the Giant Swarm App Platform.
+We fully support Helm and an app catalog is also a Helm [Chart Repository](https://helm.sh/docs/topics/chart_repository/).
+Each app catalog has its own [catalog]({{< relref "/ui-api/management-api/crd/catalogs.application.giantswarm.io.md" >}})
+CR in the management cluster.
+
+The app catalog contains a tarball for each version of an app that has been published.
+As well as an index.yaml and other metadata files that are used to generate
+metadata that is stored in the management cluster in [app catalog entry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}})
+CRs.
+
+The catalog needs to be served over HTTP and there are [multiple options](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories)
+including GitHub Pages or tools like Harbor or ChartMuseum which run in a
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog using GitHub Pages
+
+First you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this repository hosts an app catalog.
+
+Create the git repository in GitHub and enable [GitHub Pages](https://docs.github.com/en/pages/quickstart)
+for the `main` branch.
+
+Initialize the empty helm index.yaml with `helm repo index .` and commit it to
+the repository.
+
+## Pushing an app to an app catalog
+
+TODO
+
+I think we should recommend using ABS (app-build-suite) and this also means our
+metadata extensions will work.
+
+But I have some questions / concerns.
+
+- Should we link to the ABS [tutorial](https://github.com/giantswarm/app-build-suite/blob/master/docs/tutorial.md)?
+- Or should we create another docs page and link them?
+- ABS doesn't commit the files to the repo (which is intentional) but I think we need a solution for that.
+- We could suggest architect-orb but that feels weird since its mainly internal and customer may not be using
+Circle CI.
+- Should we create a simple GitHub Action for this?
```

Comment:
Sharing architect(-orb) is not a good idea, IMHO. There are already github actions for simple commits like this, I'm using one for creating PRs here: https://github.com/giantswarm/github/blob/7701fdadf34ce91d29fa370ef78546bf46c3ae2d/.github/workflows/update_pre_commit.yaml#L23
I'm sure we can find something ready for commits as well.
But that's the right direction overall, I think.

Reply:
Agree architect / orb doesn't work. I like the idea of a simple github action. If customers are using other options for CI the action still works as an example.